skbeam/core/correlation.py

```diff
@@ -121,7 +119,7 @@ def _one_time_process(buf, G, past_intensity_norm, future_intensity_norm,
 
 results = namedtuple(
     'correlation_results',
-    ['g2', 'lag_steps', 'internal_state']
+    ['correlation_results', 'lag_steps', 'internal_state']
```

Comment:
I renamed result object from g2 to correlation_results to use for both one_time and two_time